### ISSUE TITLE
GitHub Publish-Bot Failing

### ISSUE THREAD

**Issue by westonnelson** [ethereum-optimism/ethereum-optimism.github.io#274]
**Description:**
bot(ci): generate token list is failing to Publish due to missing credentials in GitHub Actions.

**Steps to reproduce:**
1. Go to [Token List Merged PR](https://github.com/ethereum-optimism/ethereum-optimism.github.io/actions/runs/4139825620/jobs/7157844908)
2. Scroll down to "Login to DockerHub"
4. See error: "Username and password required."

**Screenshots**

See screenshot of publish-bot.

![publish-bot](https://user-images.githubusercontent.com/29180454/218554116-ee011235-13de-4b3c-a445-10b560b1005f.png)

**Additional context**

The NFTEarth team is intending to bridge from Mainnet to Optimism as soon as possible. Thank you OP team!

**Comment by zainbacchus** [comment]
Hey @westonnelson were you able to unblock the NFTEarth team?

**Comment by westonnelson** [comment]
Hey @zainbacchus ! Do you mean get the token able to be bridged? If so, then yes!

The first Tx went through to Optimism from the deployed without issue - but I do see the Optimistic Etherscan details weren't synced...appear to be updated much like Etherscan so I assume that is a separate task.

Thank you for following up. 🔥

**Comment by zainbacchus** [comment]
" but I do see the Optimistic Etherscan details weren't synced." Do you mind sharing more? Link a tx hash / expectation? 
@westonnelson

**Comment by westonnelson** [comment]
Yes absolutely! So specifically to mean with the gateway/bridge directly to Optimism:

# Ethereum Mainnet token contract address: [0x0f9b80fc3c8b9123D0aEf43Df58ebDBC034A8901](https://etherscan.io/token/0x0f9b80fc3c8b9123d0aef43df58ebdbc034a8901)

# Optimism Mainnet token contract address: [0xc96f4F893286137aC17e07Ae7F217fFca5db3AB6](https://optimistic.etherscan.io/address/0xc96f4F893286137aC17e07Ae7F217fFca5db3AB6) 

**Some reference images here too.. hope this is helpful?! Let me know!**
On Etherscan:
![NFTEarthEtherscan](https://user-images.githubusercontent.com/29180454/219466712-e833a761-7bcd-4f51-96eb-5f41939e1be8.png)
Optimistic Etherscan: 
![NFTEarthOptimism](https://user-images.githubusercontent.com/29180454/219466797-edb8b1c1-53ea-4d6a-8274-4e4377423edf.png)
Tx details of bridging from L1 to L2:
![BridgeTx](https://user-images.githubusercontent.com/29180454/219466836-e95eb94f-5e82-4b2c-bd50-6b176e616b3e.png)
